helm/values.yaml

```diff
@@ -8,6 +8,10 @@ vpnClientComponentName: vpn-client
 
 nuvlaedgeVolumeHostPath: /var/lib/nuvlaedge
 
+deploymentRevHistLimit: 0
+
+deploymentSuffix: ""
```

Comment:
Because the deployments are namespaced there will be no clash with some other probable deployments and the future components must simply be given unique names. So, I don't think we need to provide the possibility to provide the suffix. Hence, I would simply remove this.

Reply:
I did it this way in case we decide in the future to add a suffix temporarily to these deployments.
But will remove the possibility.